setup.py

```diff
@@ -3,7 +3,7 @@
 
 import sklego
 
-base_packages = ["numpy>=1.15.4", "scipy>=1.2.0", "scikit-learn>=0.20.2",
+base_packages = ["numpy>=1.16.0", "scipy>=1.2.0", "scikit-learn>=0.20.2",
```

Comment:
Are there issues with numpy 1.15?

Reply:
on python3.7 -> yes. That's why I've added it.

Reply:
okay!